File: week 3 - Presentation/code examples/3 - loop/loop.java


class Test
{
    public static void main(String[] args)
    {

        // Loop 5 times
        for (int i=0;i<=5;i++)
        {
              System.out.println(Integer.toString(i));
        }

        // loop until stop is true
        Boolean stop = false;
        while (!stop)
        {
          System.out.println("Looping until stop is true");
          stop = false; // loops once because we set stop to true
        }

        // loop forever
        while (true)
        {
              System.out.println("Looping Forever");
        }


        // Do an action first and then check if it should exit the loop
        // Do while loops do not exsist in python

        int i = 0;
        do
        {
          System.out.println("Hello World");

          i++;
        }
        while (i > 5);


    }
}

File: week 3 - Presentation/code examples/3 - loop/loop.py


# Loop 5 times
for i in [0,1,2,3,4]:
    print(i);

for i in range(0,5):
    print(i);


list2 = ["Hello","World","12345678","Testing","Learning"];


for name in ["tristan","amos","Hello","world"]:
    print(name)


# loop until stop is true
stop = True;
while stop == True:
    print("Looping until stop is true")
    stop = False; # loops once because we set stop to true

# loop forever
while 1:
    print("Looping Forever")

# loop forever
while True:
    print("Looping Forever")


for filename in os.listdir("./"):
    print(" File or Folder -> " + filename)
